ceph/rados/core_workflows.py

```diff
@@ -1034,18 +1035,206 @@ def get_journalctl_log(
             log.error(f"Exception hit while command execution. {er}")
         return log_lines
 
-    def set_mclock_profile(self, profile, osd="osd"):
-        """Set OSD MClock Profile.
+    def config_param_chk(self, feature):
+        """
+        Used to check the osd configuration parameter value
+        Args:
+            feature: feature name in the ini file
+
+        Returns : True -> pass, False -> fail
+        """
+        # Read Config paramters
+        config_info = ConfigParser()
+        get_config_baseCmd = "ceph config get osd  "
+
+        config_info.read("tests/rados/rados_config_parameters.ini")
+        # Read values from mclock section
+
+        parameters = config_info.items(feature)
+        for param in parameters:
+            get_config_cmd = f"{get_config_baseCmd}{param[0]}"
+            status = self.run_ceph_command(cmd=get_config_cmd)
+            # Return if any parameter is not equal to the default value
+            if str(status).strip() != str(param[-1]).strip():
+                return False
+        # Return True if all parameters are pass
+        return True
+
+    def set_rados_flag(self, flag):
+        """
+        Used to set the rados flag
+        Args:
+            flag: Rados flag to set
+
+        Returns : True -> pass, False -> fail
+        """
+        try:
+            set_flag_cmd = f"{'ceph osd set '} {flag}"
```

Comment:
I believe `ceph osd set` is static in this f-string.

`f"ceph osd set {flag}"` If quotes needed `f"'ceph osd set' {flag}"`

Reply:
Implemented as expeceted